### ISSUE TITLE
webcamoid 9.0.0 cant add virtual camera using pkexec

### ISSUE THREAD

**Issue by mixalbl4-127** [webcamoid/webcamoid#510]
```
 [00007f96f0c822b0] main decoder error: failed to create video output
 [2022-04-13 11:03:36.017, Webcamoid, 0x7f974b9f27c0,  (0)] debug: pkexec --version |
        --help |
        --disable-internal-agent |
        [--user username] PROGRAM [ARGUMENTS...]
 
 See the pkexec manual page for more details.
```

App: webcamoid-portable-linux-9.0.0-x86_64
OS: Ubuntu 20.04
pkexec version 0.105

previous version (8.6.1) works fine with it.
fount same error: https://www.mail-archive.com/debian-bugs-dist@lists.debian.org/msg1845604.html

**Comment by hipersayanX** [comment]
Does this command works for you?

```
echo "echo Hello world" | pkexec
```

**Comment by mixalbl4-127** [comment]
@hipersayanX 
`$ echo "echo Hello world" | pkexec`
```
pkexec --version |
       --help |
       --disable-internal-agent |
       [--user username] PROGRAM [ARGUMENTS...]

See the pkexec manual page for more details.

```

**Comment by hipersayanX** [comment]
Ok, passing the commands through the stdin works perfectly fine in Arch, doesn't work in Debian derivatives, I can then write the commands to a script fine, but then Debian users will be exposed to a [security risk](https://github.com/webcamoid/webcamoid/issues/330), I can then maybe write a command just for updating the virtual camera settings and call it with pkexec, but don't expect it until maybe hopefully next year because there are no other developers working on the project. What should I do then? :angry:

**Comment by billypurdue** [comment]
> 

In the meantime is there a way we can create the virtual camera by hand?  I was poking around in the source but I'm not nearly smart enough to figure that out.  Otherwise for many this will be useless until it's resolved.

**Comment by hipersayanX** [comment]
@billypurdue yes, just check the [wiki](https://github.com/webcamoid/akvcam/wiki).

**Comment by mixalbl4-127** [comment]
```
echo "echo Hello world" | pkexec /bin/sh
Hello world
```
works fine! @hipersayanX

**Comment by hipersayanX** [comment]
You can suggest Debian team to patch [these](https://github.com/webcamoid/webcamoid/blob/3bf5c1d99a0f406be5de84a9d8bddd8ca66f4a80/libAvKys/Plugins/VirtualCamera/src/akvcam/src/vcamak.cpp#L1966) [lines](https://github.com/webcamoid/webcamoid/blob/3bf5c1d99a0f406be5de84a9d8bddd8ca66f4a80/libAvKys/Plugins/VirtualCamera/src/v4l2lb/src/vcamv4l2lb.cpp#L1204) to

```
su.start(sudoBin, QStringList {"/bin/sh"});
```

**Comment by hipersayanX** [comment]
@iam-TJ I usually prefer distros to follow upstream, but looking at your explanation, they are actually right. Anyway it was not a big deal to fix that.

**Comment by hipersayanX** [comment]
Bug fixed in Webcamoid 9.1.0.
